packages/react-aria-components/stories/index.stories.tsx

```diff
@@ -746,3 +748,186 @@ function CustomTab(props) {
+      <FileTrigger onChange={action('onChange')} >
+        <Button>Upload</Button>
+      </FileTrigger>
+    </DropZone>
+  </div>
+);
+
+export const DropzoneExampleWithDraggableAndFileTrigger = (props) => (
+  <div>
+    <Draggable />
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      <FileTrigger onChange={action('onChange')} >
+        <Button>Browse</Button>
+      </FileTrigger>
+      Or drag into here
+    </DropZone>
+  </div>
+);
+
+export const DropZoneOnlyAcceptPNGWithFileTrigger = (props) => (
+  <div>
+    <DropZone
+      {...props}
+      getDropOperation={(types) =>  types.has('image/png') ? 'copy' : 'cancel'}
+      className={styles.dropzone}
+      onPress={action('OnPress')}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')} >
+      <FileTrigger onChange={action('onChange')} accept="image/png">
+        <Button>Upload</Button>
+      </FileTrigger>
+    </DropZone>
+  </div>
+);
+
+export const DropzoneExampleWithDraggableObject = (props) => (
+  <div>
+    <Draggable />
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')} >
+      <Text slot="heading">
+        DropZone Area
+      </Text>
+    </DropZone>
+  </div>
+);
+
+export const DropzoneExampleWithCopyableObject = (props) => (
+  <div>
+    <Copyable />
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      <Text slot="heading">
+        DropZone Area
+      </Text>
+    </DropZone>
+  </div>
+);
+
+export const DropzoneWithRenderProps = (props) => (
+  <div>
+    <Draggable />
+    <Copyable />
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onPress={action('OnPress')}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      {({isHovered, isFocused, isFocusVisible, isDropTarget}) => (
+        <div>
+          <Text slot="heading">
+            DropzoneArea
+          </Text>
+          <div>isHovered: {isHovered ? 'true' : 'false'}</div>
+          <div>isFocused: {isFocused ? 'true' : 'false'}</div>
+          <div>isFocusVisible: {isFocusVisible ? 'true' : 'false'}</div>
+          <div>isDropTarget: {isDropTarget ? 'true' : 'false'}</div>
+        </div>
+      )}
+    </DropZone>
+  </div>
+);
+
+export const FileTriggerButton = (props) => (
+  <FileTrigger 
+    {...props} 
+    onChange={action('OnChange')} >
+    <Button>Upload</Button>
+  </FileTrigger>
+);
+
+export const FileTriggerLink = (props) => (
+  <FileTrigger 
+    {...props} 
+    onChange={action('OnChange')}
+    multiple >
```

Comment:
Doesn't seem like this story accepts multiple file, looks like the `multiple` isn't being propagated to the `input`

Reply:
oops i forgot to update the story...that's my bad